src/key.cpp

```diff
@@ -22,7 +22,8 @@
 
 #include "header.h"
 
-char pCharset[] = "BCDFGHJKMPQRTVWXY2346789";
+/* The allowed character set in a product key. */
+char pKeyCharset[25] = "BCDFGHJKMPQRTVWXY2346789";
```

Comment:
I don't think 25 was necessary here. It only adds ambiguity, since the product key is 25 characters long.
The character set is 24 long, and a null terminator isn't helping with understanding